platform/security/certificate/security-ocsp-checker/src/main/java/org/codice/ddf/security/ocsp/checker/OcspChecker.java

```diff
@@ -105,14 +111,21 @@ public boolean passesOcspCheck(X509Certificate[] certs) {
       try {
         Certificate certificate = convertToBouncyCastleCert(cert);
         OCSPReq ocspRequest = generateOcspRequest(certificate);
-        List<OCSPResp> ocspResponses = sendOcspRequests(cert, ocspRequest);
-        if (anyOcspResponseStatusIsRevoked(ocspResponses)) {
+        Map<String, CertificateStatus> ocspStatuses = sendOcspRequests(cert, ocspRequest);
+        String revokedStatusUrl = getFirstRevokedStatusUrl(ocspStatuses);
+        if (revokedStatusUrl != null) {
           SecurityLogger.audit(
-              "Certificate {} has been revoked by the OCSP server {}.", cert, ocspServerUrls);
-          LOGGER.debug("Certificate has been revoked by the OCSP server.");
+              "Certificate {} has been revoked by the OCSP server at URL {}.",
+              cert,
+              revokedStatusUrl);
+          LOGGER.warn(
+              "Certificate {} has been revoked by the OCSP server at URL {}.",
+              cert,
+              revokedStatusUrl);
           return false;
         }
-      } catch (OcspCheckerException ignore) {
+      } catch (OcspCheckerException e) {
+        postErrorEvent(e.getMessage());
```

Comment:
📘 This is the only place where `postErrorEvent` is called, since all exceptions within the try will bubble out to this point.